tests/ui/js/block-timeBasedTests.js

```diff
@@ -87,70 +86,6 @@ https://raw.githubusercontent.com/fluid-project/sjrk-story-telling/master/LICENS
                 {
                     funcName: "jqUnit.assertEquals", // 0000
                     args: ["Combined text is as expected", "", "{timeBased}.model.contentString"]
-                },
-                {
-                    func: "{timeBased}.applier.change",
-                    args: ["transcript", "Rootbeer: krrraow"]
-                },
-                {
-                    funcName: "jqUnit.assertEquals", // 0001
-                    args: ["Combined text is as expected", "Rootbeer: krrraow", "{timeBased}.model.contentString"]
-                },
-                {
-                    func: "{timeBased}.applier.change",
```

Comment:
What gets removed is "transcript" field but not other fields. What is the idea of removing non-nullable value tests for other fields. Once these tests are removed, this test file only has tests for null field values, which doesn't seem to be what the change is for.

Reply:
The purpose of this test sequence is to test the model relay that combines the various values in the block into a string. There were previously four values being combined, so I was testing (and numbering) every possible combination, but since we've removed one of the values, half of the tests are no longer useful and so I've removed them. The remaining 8 test the various combinations of the other three values.

Reply:
Just find you've done the digit change. Thanks.